src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Server/Readme.txt

```diff
@@ -1,4 +1,4 @@
 This code includes a dependency on Duende IdentityServer.
-This is an open source product with a reciprocal license agreement. If you plan to use IdentityServer in production this may require a license fee.
-To see how to use AAD for your identity please see https://aka.ms/aspnetidentityserver
+This is an open source product with a reciprocal license agreement. If you plan to use Duende IdentityServer in production this may require a license fee.
+To see how to use Azure ActiveDirectory for your identity please see https://aka.ms/aspnetidentityserver
```

Comment:
Azure Active Directory (3 separate words) 

That aka.ms link is still weirdly named, because it's positioning points it to instructions on AAD, but it has identity in the name. I can't remember what the darned link was for now, it's still a placeholder.

Reply:
Updated, look good now @blowdart ?